tests/integration/test_py.py

```diff
@@ -142,6 +142,7 @@ def foo(q1: qubit, q2: qubit, q3: qubit) -> tuple[qubit, qubit, qubit]:
     validate(module.compile())
 
 
+@pytest.mark.skip("Requires Tket2 upgrade")
```

Comment:
Pytket measurements in tket2 now returns a custom `Bit` type instead of the prelude bool. This means, we need to load the Tket1 extension for validation, but the extension is still at Hugr v0.3

Reply:
Tket2 is on the Hugr 0.4 release candidate now, but it still doesn't work. We need to capture the `LBit` type or write a conversion pass that turns TKET1 measurements into TKET2 measurements...